xbrowse/datastore/elasticsearch_datastore.py

```diff
@@ -162,13 +171,18 @@ def get_elasticsearch_variants(self, project_id, family_id=None, variant_filter=
             # TODO add caching
             indiv_id = _encode_name(family_individual_ids[0])
 
-            mapping = self._es_client.indices.get_mapping(str(elasticsearch_index)+"*")
+
+            matching_indices = []
+            mapping = self._es_client.indices.get_mapping(str(elasticsearch_index)+"*,rare_genomes_project_wgs_rgp_*")
```

Comment:
why are we hardcoding RGP here?

Reply:
that was a test for adding SVs, wasn't meant to be committed. Will take it out.